File: Web/Khushi-Vaish_KhushiVaish_2024cs1147_2/GettingStarted/gradingSystem.java
import java.util.*;
public class Main{
  
  public static void main(String[] args) {
      Scanner scn=new Scanner
      (System.in);
      int marks = scn.nextInt();
      if(marks>90)
      {
          System.out.print("excellent");
      }
      else if(marks<=90 && marks>80)
      {
          System.out.print("good");
      }
      else if(marks<=80 && marks>70)
      {
          System.out.print("fair");
      }
      else if(marks<=70 && marks>60)
      {
          System.out.print("meets expectations");
      }
      else if(marks<=60)
      {
         System.out.println("below par");
      }
      }
  }

  
   

File: AR-VR/vanshverma721_VanshVerma_2024csit1153_2/GettingStarted/gradingsystem.c
#include<iostream>
using namespace std;
int main()
{
     int marks;
    scanf("%d",&marks);
 if(marks>90)
 {
     printf("excellent");
 }
 
 if(marks>80 && marks<=90)
 {
     printf("good");
 }
 
 if(marks>70 && marks<=80)
 {
     printf("fair");
 }
 
  if(marks>60 && marks<=70)
 {
     printf("meets expectations");
 }
 
  if(marks<60)
 {
     printf("below par");
 }
 
 return 0;
 
}